CODEOWNERS

```diff
@@ -3,9 +3,13 @@
 # Server
 /codegen-server-test/                           @awslabs/smithy-rs-server
 /codegen-server/                                @awslabs/smithy-rs-server
-/rust-runtime/aws-smithy-http-server-python/    @awslabs/smithy-rs-server
 /rust-runtime/aws-smithy-http-server/           @awslabs/smithy-rs-server
 
+# Python Server
+/codegen-server-test/python/                    @awslabs/smithy-rs-python-server
+/codegen-server/python/                         @awslabs/smithy-rs-python-server
+/rust-runtime/aws-smithy-http-server-python/    @awslabs/smithy-rs-python-server
```

Comment:
I think we want to keep the Rust server people as owners of these folders as well and not just have the Python server.